ncm-spma/src/main/pan/components/spma/yumng/schema.pan

```diff
@@ -66,6 +66,10 @@ type component_spma_yum = {
     "quattor_os_release" ? string # string to write to quattor_os_file
     "plugins" ? spma_yum_plugins
     "main_options" ? spma_yum_main_options
+    @{ Run the SPMA after configuring it }
+    "run" ? legacy_binary_affirmation_string
+    @{ Allow user installed (i.e. unmanaged) packages to be present on the system }
+    "userpkgs" ? legacy_binary_affirmation_string
 };
 
 bind "/software/components/spma" = component_spma_yum;
```

Comment:
I've been working on cleaning up the schema in a seperate branch, I think this would be better fixed there.